lib/matplotlib/tests/test_colorbar.py

```diff
@@ -514,8 +514,29 @@ def test_colorbar_scale_reset():
 def test_colorbar_get_ticks():
     with rc_context({'_internal.classic_mode': False}):
 
-        fig, ax = plt. subplots()
+        fig, ax = plt.subplots()
         np.random.seed(19680801)
         pc = ax.pcolormesh(np.random.rand(30, 30))
         cb = fig.colorbar(pc)
         np.testing.assert_allclose(cb.get_ticks(), [0.2, 0.4, 0.6, 0.8])
+
+
+def test_colorbar_inverted_ticks():
+    fig, axs = plt.subplots(2, 1)
+    axs = axs.flat
```

Comment:
If you eliminate the "1" in the subplots call, then you don't need the "flat" line.

Reply:
learn something new every day!